fit_multiprofit_config.py

```diff
@@ -0,0 +1,12 @@
+from modelling_research.multiprofit_task import MultiProFitTask
+
+config.connections.name_output_cat = "multiprofit"
+config.fit_multiband.retarget(MultiProFitTask)
+config.fit_multiband.bands_read = ["r"]
+config.fit_multiband.fitCModelExp = False
+config.fit_multiband.fitSersicFromCModel = True
+config.fit_multiband.computeMeasModelfitLikelihood = True
+config.fit_multiband.usePriorShapeDefault = True
+config.fit_multiband.priorCentroidSigma = 0.2
+config.fit_multiband.priorMagBand = "r"
+config.fit_multiband.psfHwhmShrink = 0.1
```

Comment:
Should these defaults just live in `MultiProFitTask`'s config class?

Reply:
They already have defaults; these are my currently-favoured overrides that I'm not confident enough to make into defaults.

I'm not sure if it's generally a good idea to have a default prior band, although in this case I haven't actually implemented a prior based on any band besides `r` (yet).